packages/internal/src/callback.js

```diff
@@ -115,6 +126,9 @@ harden(makeSyncMethodCallback);
  * @returns {Callback<I>}
  */
 export const makeMethodCallback = (target, methodName, ...bound) => {
+  isObject(target) || Fail`method callback target must be an object: ${target}`;
+  typeof methodName === 'string' ||
```

Comment:
Actually thinking about this, I believe we should allow well-known symbols since they're allowed as method names in `E`. cc @michaelfig